modules/3d/LICENSE_KinectFusion.md

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2012, Anatoly Baksheev
```

Comment:
> KinectFusion.md

should left in opencv_contrib.

Reply:
Move this line to relevant .cpp files only.

Reply:
Forgot to delete this file, now fixed